sui/deploy-contract.js

```diff
@@ -92,18 +117,20 @@ async function postDeployTest(published, config, chain, options) {
     printInfo('Register transaction', registerTx.digest);
 }
 
-async function postDeployOperators(published, chain) {
+async function postDeployOperators(published, args) {
+    const { chain } = args;
     const [operatorsObjectId, ownerCapObjectId] = getObjectIdsByObjectTypes(published.publishTxn, [
         `${published.packageId}::operators::Operators`,
         `${published.packageId}::operators::OwnerCap`,
     ]);
     chain.contracts.Operators.objects = {
-      Operators: operatorsObjectId,
-      OwnerCap: ownerCapObjectId,
+        Operators: operatorsObjectId,
+        OwnerCap: ownerCapObjectId,
     };
 }
 
-async function postDeployAxelarGateway(published, keypair, client, config, chain, options) {
+async function postDeployAxelarGateway(published, args) {
+    const { keypair, client, config, chain, options } = args;
```

Comment:
pass all the same args to each post deploy method instead of wrapping it inside args?

Reply:
will do. i just don't want to declare a function with long positional args, but actually we did that everywhere across the codebase 😂